diktat-rules/src/test/resources/test/paragraph3/newlines/ListArgumentLambdaTest.kt

```diff
@@ -0,0 +1,10 @@
+package test.paragraph3.newlines
+
+class Example {
+
+    fun foo() {
+        foldIndexed("#") { index: Int, acc: String, pathPart: String ->
```

Comment:
I believe this case should be an exception: we should either allow multiple args on the same line, or move the first too. Let's discuss. Also, please add a test with 3 args and no explicit types.